### ISSUE TITLE
Please support dockerimage with package manager installed

### ISSUE THREAD

**Issue by m00lecule** [GoogleContainerTools/kaniko#2475]
**Actual behavior**
Hi,

I would kindly request another container images branch with common package manager preinstalled - for instance `apk`.

Currently I am struggling to prepare custom Dockerfile based on `gcr.io/kaniko-project/executor:debug` that has also python installed. I have tried copying the kaniko executable to `alpine` container image but it made kaniko builds unstable.

**Expected behavior**

Ability to run `apk add --no-cache curl` in community supported Dockerimage

For example: 

```dockerfile
FROM gcr.io/kaniko-project/executor:v1.9.2-alpine
RUN apk add --no-cache python3 py3-pip
```

**Comment by oallauddin** [comment]
Yes please switch from busybox to alpine. I have been fighting with the version of wget that comes with busybox. I can't get it to work in our environment. does not work with https sites and a proxy. busybox is single binary representing many programs and have been unsuccessful in overriding wget.
